rq/worker.py

```diff
@@ -713,6 +713,7 @@ def monitor_work_horse(self, job, queue):
 
         ret_val = None
         job.started_at = utcnow()
+        job.heartbeat(job.started_at)
```

Comment:
The first heartbeat should be located in [prepare_job_execution()](https://github.com/rq/rq/blob/master/rq/worker.py#L825) so it can be pipelined with job status changes etc.

Reply:
This is no longer needed since the first heartbeat is already set in `prepare_job_execution()`.

Reply:
Don't forget to remove this line

Reply:
thx, done, sorry for being messy